R/bins_to_bedgraph.R

```diff
@@ -1,3 +1,13 @@
+#' @title Bins to bed graph.
+#'
+#' @description This function will generate bed file.
+#'
+#' @param bin_df Data frame with bins.
+#' @param min_value Minimum value to retain the bin.
+#' @param filename Path to a local file on drive to save the resulting file.
+#'
```

Comment:
Maybe we should say "nothing" for the return or (better yet) make functions like this one return the "full path to the file that was written"

Reply:
Agree, added this to this function and other similar functions that were writing files